h2o-r/h2o-package/R/grid.R

```diff
@@ -228,3 +234,90 @@ h2o.getGrid <- function(grid_id, sort_by, decreasing) {
       summary_table     = json$summary_table
       )
 }
+
```

Comment:
same comment as for Python.
R has 2 modules `import` and `export`, those functions are probably more suitable there for consistency.
Also, it would prefer having naming and signature symmetric with models: `h2o.loadGrid`, `h2o.saveGrid`.

Reply:
Renamed the methods for now. Will deal with the signature later.

@ledell Do we want the tidyverse version, or the "traditional H2O" with camelCase ?

Reply:
Done. Re-aligned with older methods using camelCase, even though we're only doing snake_case now. Makes sense to me - would be too much entropy for the users.

Moved to import/export. Changed the examples accordingly.